src/BookNavigator/search/search-provider.js

```diff
@@ -51,6 +53,25 @@ export default class {
     window.addEventListener('BookReader:SearchCallbackNotIndexed', (event) => { this.onSearchRequestError(event, 'notIndexed') });
     window.addEventListener('BookReader:SearchCallbackError', (event) => { this.onSearchRequestError(event) });
     window.addEventListener('BookReader:SearchResultsCleared', () => { this.onSearchResultsCleared() });
+    window.addEventListener('BookReader:SearchCanceled', (e) => { this.onSearchCanceled(e) });
+  }
+
+  /**
+   * Cancel search handler
+   * resets `searchState`
+   */
+  onSearchCanceled() {
+    searchState = {
```

Comment:
_Question_ Where is `searchState` defined?

Reply:
it's at the top of the file:
![image](https://user-images.githubusercontent.com/7840857/119871979-d3777700-bed7-11eb-98b9-14468939685a.png)